### DISCUSSION TITLE
3rd party memory cards at fault for inability to exploit SCPH-5501?

### DISCUSSION THREAD

**Discussion by jeffbrl** [brad-lin/FreePSXBoot#71] [General]
I have an NTSC-U SCPH-5501 with BIOS 11/18/96 firmware (verified using Unirom). I'm using MCA to write the image to 3rd party memory cards. I am unable to trigger the exploit using the slot 1 or slot 2 images. I have tried writing using the PS2 slim and PS2 phat.

One strange thing I noticed is that I can rewrite the memory cards using MCA. I expected a need to perform additional steps to zero out the card between MCA restore operations given the intentional corruption mentioned in the README.

FWIW, I've also tried writing tonyhax images to the memory cards. This doesn't work either. I am able to trigger the exploit using tonyhax with Crash Bandicoot 2.

Has anyone encountered problems using the cheap memory cards from Aliexpress?

**Comment by brad-lin** [comment]
Some 3rd party memory cards are faulty, some do not even have 128 KB capacity but 64 instead.

Your best bet since you can run Unirom already would be to try with the Unirom Boot CD: https://github.com/JonathanDotCel/unirom8_bootdisc_and_firmware_for_ps1/releases/tag/8.0.J

**Comment by jeffbrl** [comment]
@brad-lin Thank you for the tip. Unirom produced an error message that I was ripped off as my memory card doesn't have the capacity it should. I'll buy a new memory card.

**Reply by nicolasnoble** [reply]
If you open it, you should see it's got a 512kBits eeprom chip instead of a 1MBits one. That'll confirm it's a fake card.

Like this one:

![image](https://user-images.githubusercontent.com/7281574/136139901-82c86511-1785-4aaf-9370-24ab28132a06.png)

This one says EE512 instead of EE010, which is the wrong eeprom type.

**Reply by JonathanDotCel** [reply]
Shame you were ripped off, but it's nice to see the detection is working at least.
Would you be able to grab a picture for us, from roughly the same angle as @nicolasnoble?

**Reply by jeffbrl** [reply]
Same chip in mine.

![PXL_20211006_122934850](https://user-images.githubusercontent.com/1028225/136267207-e530ed61-f5aa-46c2-a302-adec206d9440.jpg)

**Reply by nicolasnoble** [reply]
Right so you absolutely do not want to use this card. It'll corrupt its own data if you have more than 7 blocks saved on it.

**Comment by NuSynth** [comment]
![card](https://user-images.githubusercontent.com/38872463/150396961-47a80098-e19a-492e-90e9-f475c6d252aa.jpg)
Any idea why mine doesn't work with FreePSXBoot? I set up the card reader/writer thing with my Arduino. I can write the image files to it, format it, and read it. I even checked to see if I broke something in it by saving my game to it on my playstation after I formatted it with MemcardRex. The card works. I don't know who made it, because it was in just some generic shell. But I looked up the Atmel chip. It looks like it's a good chip as far as I could tell, which isn't very far since I know next to nothing about electronics engineering.

**Reply by NuSynth** [reply]
It is a little slower at being recognized by the playstation than my other cards. Would that have anything to do with it? It doesn't use the 7.6v third pin. So I thought maybe that had something to do with it being slower.

**Comment by nicolasnoble** [comment]
What happens exactly? Are you sure you've selected the proper image?

**Reply by NuSynth** [reply]
Oh yeah, sorry I forgot to add that detail. The plastic shell says it's SCPH 5501. So that's the images I tried. I tried with the slot 1 and slot 2 images, with the card in the correct slots. It doesn't do anything. It looks like its an empty memory card when I look in the memory card viewer on the playstation. I waited a long while several times while attempting to get this to work. But memcardRex shows that it's on the card when I go back to check that it's actually on the card though.

If I take off the metal thing above the motherboard, will it be hard to identify the model? While the shell does say that it's 5501, I just got it off of ebay. I really don't know if it's what the shell says it is. The screws look like someone opened it before, and I had to push the controller brick thing's connector into the board better, because it wasn't detecting that at first. I don't know why someone would have switched it out, but I'll check the board if it will tell me anything.

**Reply by NuSynth** [reply]
Correction: It looks like it's no memory card in the playstation's memory card viewer. It wouldn't let me copy any saves to that card while I had the freePSXboot images on it. I understand that I shouldn't be able to do anything with it like that with the image on it, which is why I added this comment.

**Reply by brad-lin** [reply]
Make sure you have disabled the option "Attempt to fix corrupted memory cards" (or similar, I don't remember the exact text) in MemcardRex, then write again the image.

**Reply by NuSynth** [reply]
> Make sure you have disabled the option "Attempt to fix corrupted memory cards" (or similar, I don't remember the exact text) in MemcardRex, then write again the image.

Oh thanks! I know I had that option checked! I'll be back to let you know if it works.

**Reply by NuSynth** [reply]
> Make sure you have disabled the option "Attempt to fix corrupted memory cards" (or similar, I don't remember the exact text) in MemcardRex, then write again the image.

I haven't tried slot 1's image yet this time. But all I am getting with the slot 2 image is the cyan colored screen.

**Reply by brad-lin** [reply]
Which means the exploit is working. Is the cyan getting filled by green or not? If not, your memory card might be too slow indeed.

**Reply by NuSynth** [reply]
![IMG_20220120_161931_371](https://user-images.githubusercontent.com/38872463/150424074-9ea0f7a4-5fc3-401b-9066-90e1db4481a7.jpg)

I can't really tell if it's Cyan or green. But I'm guessing it's too slow.

**Reply by NuSynth** [reply]
well it definitely looks more green than that in person. So it's too slow.

**Comment by nicolasnoble** [comment]
It could be, but you're still not describing the symptoms.
